Makefile

```diff
@@ -27,10 +27,6 @@ build: export SPDK_GIT_BRANCH != git -C spdk name-rev --name-only HEAD
 build: export SPDK_GIT_COMMIT != git rev-parse HEAD:spdk
 build: export BUILD_DATE != date -u +"%Y-%m-%dT%H:%M:%SZ"
 
-up: SVC = nvmeof ## Services
-up: OPTS ?= --abort-on-container-exit
-up: override OPTS += --no-build --remove-orphans --scale nvmeof=$(SCALE)
```

Comment:
Why this?

* `--no-build` is to force explicit building of images, which allows users to decide whether they want to pull or build them.
* `--remove-orphans` is just a clean-up flag, but if not bringing any benefits and complicating parallel deployments, then I guess it's ok to remove.
* `--scale` allows to deploy multiple gateways in a single host. It's not working yet due to the way listener IP is specified and some limitations in SPDK, but should be the way to test multi-gateway.

Reply:
It would be nice to `up` ceph cluster for pytests see [here](https://github.com/baum/ceph-nvmeof/blob/d218bb3e57661b82c8f1a1c9d0da5d2d7ce59ea7/.github/workflows/pytest.yml#L28-L31), overriding OPTS, wdyt?

Reply:
@epuertat reshuffled makefile definitions, let me know how it looks now.